pkg/gofr/datasource/file/ftp/file.go

```diff
@@ -14,8 +14,6 @@ import (
 	"time"
 
 	"github.com/jlaffaye/ftp"
-
-	file_interface "gofr.dev/pkg/gofr/datasource/file"
```

Comment:
Is this file still needed then?

Reply:
no with the updated implementation that's not needed